manifests/base/terraform-applier.yaml

```diff
@@ -52,6 +52,10 @@ apiVersion: apps/v1
 kind: StatefulSet
 metadata:
   name: terraform-applier
+  annotations:
+    "app.uw.systems/description": "Heavily adapted from kube-applier, terraform-applier enables continuous deployment of Terraform code by applying modules from a Git repository or local directory."
+    "app.uw.systems/tier": "tier_3"
```

Comment:
What is tier 3?

Reply:
The tiers are documented here: https://wiki.uw.systems/posts/opslevel-m1sicdbi#huakw-what-service-tiers-are-defined-and-what-is-their-meaning.
With this migration I'm just picking up whatever is set in Opslevel. This was picked up from https://app.opslevel.com/services/terraform-applier_3